Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: setup
+setup:
+	python3 -m venv venv && source venv/bin/activate && pip install -r requirements.txt
+
+.PHONY: lint
+lint:
+	flake8
```

Comment:
Maybe add pylint here also explicitly.

Reply:
Not used much, tends to be very verbose, but will try again ... thanks.